src/bootstrap-table.js

```diff
@@ -3190,15 +3190,15 @@ class BootstrapTable {
       this.options.escape ? Utils.escapeHTML(params.title) : params.title
 
     if (this.columns[this.fieldsColumnsIndex[params.field]].visible) {
-      const header = this.options.height !== undefined ? this.$tableHeader : this.$header
-
-      header.find('th[data-field]').each((i, el) => {
+      this.$header.find('th[data-field]').each((i, el) => {
```

Comment:
What happen if the `data-height` is set?

Reply:
I have updated to `this.resetView()` to solve the problem.